tests/Transports.AspNetCore.Tests/NewtonsoftJsonTests.cs

```diff
@@ -41,64 +41,57 @@ public async Task Decodes_BigInteger()
             var request = @"{""variables"":{""a"":1234567890123456789012345678901234567890}}";
             var ret = await Deserialize(request);
             var bi = BigInteger.Parse("1234567890123456789012345678901234567890");
-            ret.Single.Inputs["a"].ShouldBeOfType<BigInteger>().ShouldBe(bi);
+            ret.Single().Variables["a"].ShouldBeOfType<BigInteger>().ShouldBe(bi);
         }
 
         [Fact]
         public async Task Dates_Should_Parse_As_Text()
         {
             var ret = await Deserialize(@"{""variables"":{""date"":""2015-12-22T10:10:10+03:00""}}");
-            ret.Single.Inputs["date"].ShouldBeOfType<string>().ShouldBe("2015-12-22T10:10:10+03:00");
+            ret.Single().Variables["date"].ShouldBeOfType<string>().ShouldBe("2015-12-22T10:10:10+03:00");
         }
```

Comment:
This is not the default behavior by the Newtonsoft.Json serializer.

Reply:
_not_ or _now_ ?